src/Validation/BrValidation.php

```diff
@@ -82,10 +82,14 @@ public static function cpf($cpf = null){
             '99999999999');
 
         // Check number size, it pass:
-        if (strlen($cpf) != 11 || in_array($cpf, $invalid_cpfs)) {
+        if (strlen($cpf) != 11){
+            return false;
+        }else if(in_array($cpf, $invalid_cpfs)) {
             return false;
          // Calculate check digits:
-         } else {        
+         } else {
+            $d = 0;
```

Comment:
basically this is the only fix, right? so the diff should only show this line changed, not the noise around (which are all invalid changes it seems).